### PR TITLE
group services w same selectors in `oc status`

### PR DESCRIPTION
**Pull request by juanvallejo**
Fixes https://github.com/openshift/origin/issues/7503

**Before**
```bash
$ oc status
In project My Project (myproject) on server https://127.0.0.1:8443

svc/foo - foo.com
  dc/myapp deploys istag/myapp:latest <-
    bc/myapp source builds https://github.com/juanvallejo/myapp on openshift/nodejs:8
    deployment #1 deployed 7 minutes ago - 1 pod

svc/bar - 172.30.75.107:8080
  dc/myapp deploys istag/myapp:latest <-
    bc/myapp source builds https://github.com/juanvallejo/myapp on openshift/nodejs:8
    deployment #1 deployed 7 minutes ago - 1 pod
...
```

**After**
```bash
$ oc status
In project My Project (myproject) on server https://127.0.0.1:8443

svc/bar - 172.30.75.107:8080
svc/foo - foo.com
  dc/myapp deploys istag/myapp:latest <-
    bc/myapp source builds https://github.com/juanvallejo/myapp on openshift/nodejs:8
    deployment #1 deployed 7 hours ago - 1 pod
...
```

cc @soltysh @deads2k

### CODE REVIEW

**Comment by juanvallejo** [comment]
cc @soltysh

File: pkg/oc/cli/describe/projectstatus.go [hunk 1]
```diff
@@ -164,6 +164,37 @@ func (d *ProjectStatusDescriber) MakeGraph(namespace string) (osgraph.Graph, set
 	return g, forbiddenResources, nil
 }
 
+func selectorEquals(A, B map[string]string) bool {
```
Comment (soltysh) [inline pkg/oc/cli/describe/projectstatus.go]: Have you looked at `k8s.io/kubernetes/staging/src/k8s.io/apimachinery/pkg/labels` to compare selectors and labels? I think you could reuse some of the logic from there.

**Comment by juanvallejo** [comment]
@soltysh thanks, updated to use k8s.io/apimachinery/pkg/labels

**Pull request merged**